src/SonarQube.Client/Api/V9_6/GetRulesWithEducationPrinciplesRequest.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using SonarQube.Client.Api.V5_50;
+using SonarQube.Client.Logging;
+using SonarQube.Client.Models;
+
+namespace SonarQube.Client.Api.V9_6
+{
+    public class GetRulesWithEducationPrinciplesRequest : IGetRulesRequest
+    {
+        private readonly GetRulesRequest innerRequest = new GetRulesRequest();
```

Comment:
Could use v9.5 here instead of v5, less stuff to modify. Inheritance could work here. It has it's drawbacks (for example, if we in the future would need to remove a field instead of adding), but it's unlikely it'll cause problems or we can deal with them later IMO